docs/src/templates/pages/documentation/css/fozzie.hbs

```diff
@@ -45,3 +45,63 @@ When your Sass is compiled, it will include the Sass from the fozzie module as w
 If you want to check what `@import` keyword you need to include to import a module, you can check out the [Eyeglass config](https://github.com/justeat/fozzie/blob/master/package.json#L51-L60) which is located in each modules `package.json`.
 
 
+<a name="themes"></a><h2 class="sg-sectionHeading">Applying Themes (i.e. Menulog)</h2>
+
+Fozzie and it's associated components have been setup to allow for multiple themes to be generated from the same base set of styles.
```

Comment:
Apostrophe police alert! You don't need the apostrophe here - "Fozzie and its...."